apps/src/templates/studioHomepages/twoColumnActionBlock.module.scss

```diff
@@ -22,7 +22,6 @@ $breakpoint: 993px;
   .image {
     margin: 1.25rem;
     width: calc(50% - 1.25rem);
-    height: fit-content;
```

Comment:
omg good find. Such a frustrating bug!

Reply:
Ack I know!! I forgot to cross-browser check 🙃